api/v1/controllers/subjects.js

```diff
@@ -33,6 +34,9 @@ const apiErrors = require('../apiErrors');
 const redisSubjectModel = require('../../../cache/models/subject');
 const sampleStoreFeature =
                   require('../../../cache/sampleStore').constants.featureName;
+const jobType = require('../../../jobQueue/setup').jobType;
+const jobWrapper = require('../../../jobQueue/jobWrapper');
+const WORKER_TTL = 30000;
 const ZERO = 0;
```

Comment:
My understanding was that the 30 sec timeout is specific to the getHierarchy job. Would it be the same for future API jobs?
@iamigo